py/strato/common/log/log2text.py

```diff
@@ -56,25 +65,62 @@ def __init__(self, relativeTime, withThreads, showFullPaths, noDebug, microsecon
             (NORMAL_COLOR if useColors else '') + \
             ("(%(pathname)s:%(lineno)s)" if showFullPaths else "(%(module)s::%(funcName)s:%(lineno)s)")
 
-    def process(self, obj):
-        if obj['levelno'] < self._minimumLevel:
+    def process(self, line, logPath, logConf):
+        if os.path.basename(logPath).endswith(".stratolog"):
+            return self._processStratolog(line)
+        elif os.path.basename(logPath) == "exceptions.log":
+            return self._processExceptionLog(line)
+        return self._processGenericLog(line, logConf)
+
+    def _getLogTypeConf(self, logPath):
```

Comment:
calling it outside the process function means calling it once per file (except for exception.log). calling it inside process means running it once per line